ldap-api/src/main/java/ca/bc/gov/hlth/ldapapi/controller/UsersController.java

```diff
@@ -1,7 +1,7 @@
 package ca.bc.gov.hlth.ldapapi.controller;
 
 import ca.bc.gov.hlth.ldapapi.model.UserCredentials;
-import ca.bc.gov.hlth.ldapapi.service.LdapService;
+import ca.bc.gov.hlth.ldapapi.service.UserService;
```

Comment:
I split `LdapService` into two classes: `LdapService` and `UserService`. I didn't rewrite the functionality, but just moved the method around.